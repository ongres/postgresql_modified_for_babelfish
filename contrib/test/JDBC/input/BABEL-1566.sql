-- Test CHECKSUM function works on string input
select CHECKSUM('abcd');
go

-- Test CHECKSUM function works on scalar input
select CHECKSUM(123);
go

select CHECKSUM(10.12345);
go

-- Test CHECKSUM works on table column
create table t1 (a int, b varchar(10));
insert into t1 values (12345, 'abcd');
insert into t1 values (12345, 'abcd');
insert into t1 values (23456, 'bcd');
go

select CHECKSUM(a), CHECKSUM(b) from t1;
go

-- clean up
drop table t1;
go
