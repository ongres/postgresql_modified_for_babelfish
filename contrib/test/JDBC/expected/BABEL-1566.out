-- Test CHECKSUM function works on string input
select CHECKSUM('abcd');
go
~~START~~
int
-486772404
~~END~~


-- Test CHECKSUM function works on scalar input
select CHECKSUM(123);
go
~~START~~
int
539801954
~~END~~


select CHECKSUM(10.12345);
go
~~START~~
int
819228203
~~END~~


-- Test CHECKSUM works on table column
create table t1 (a int, b varchar(10));
insert into t1 values (12345, 'abcd');
insert into t1 values (12345, 'abcd');
insert into t1 values (23456, 'bcd');
go
~~ROW COUNT: 1~~

~~ROW COUNT: 1~~

~~ROW COUNT: 1~~


select CHECKSUM(a), CHECKSUM(b) from t1;
go
~~START~~
int#!#int
-2105750770#!#-486772404
-2105750770#!#-486772404
-1379210184#!#-726154620
~~END~~


-- clean up
drop table t1;
go
